cove_project/settings.py

```diff
@@ -25,18 +25,10 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
 
-
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
-
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '77mjwmr6u9n%r%h12ge90x!$&$441c=y!#$(*(g!z-75##fz+z'
-
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-
-ALLOWED_HOSTS = []
-
+DEALER_TYPE = settings.DEALER_TYPE
+SECRET_KEY = settings.SECRET_KEY
+DEBUG = settings.DEBUG
+ALLOWED_HOSTS = settings.ALLOWED_HOSTS
```

Comment:
Hmm. This doesn't seem to allow env vars to be set on the cmdline and work their way through to the application. When I run it with:

`ALLOWED_HOSTS='192.168.88.243' python manage.py runserver 0.0.0.0:8000`

I get

```
DisallowedHost at /
Invalid HTTP_HOST header: '192.168.88.243:8000'. You may need to add '192.168.88.243' to ALLOWED_HOSTS.
```

Is this expected behaviour? This is with me running it on a different machine to the one I'm trying to access it on.

Reply:
Should do, I've had this working on the live and my dev server. I'll look in a minute.

Reply:
This works fine for me

    ALLOWED_HOSTS="127.0.0.1" python manage.py runserver 0:5000

Go to http://127.0.0.1:5000/ - loads - Go to http://localhost:5000/ - fails. (And this is technically on different machines, I'm using a WM to get the right version of Python)